src/components/OttItem.js

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { makeStyles } from '@mui/styles';
+import { Avatar, Box, Typography } from '@mui/material';
+
+const ottImage = {
+  1: 'netflix',
+  2: 'disneyPlus',
+  3: 'wave',
+  4: 'watcha',
+};
+
+const Button = styled.button`
+  background-color: white;
+  border: 0px;
+`;
+
+const useStyles = makeStyles({
+  avatar: {
+    filter: 'grayscale(100%)',
+    cursor: 'pointer',
+  },
+  selectedAvatar: {
+    filter: 'grayscale(0%)',
+    cursor: 'pointer',
+  },
+});
+
+const OttItem = ({ ottId, ottName, selectedId, onSelectOtt }) => {
+  const classes = useStyles();
+
+  const handleClickOtt = () => {
+    onSelectOtt(ottId);
+  };
+
+  return (
+    <Button>
+      <Box
+        className={
+          ottId === selectedId ? classes.selectedAvatar : classes.avatar
+        }
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          flexDirection: 'column',
+          p: 1,
+          m: 1,
+        }}
```

Comment:
```suggestion
        sx={{
          display: 'flex',
          alignItems: 'center',
          flexDirection: 'column',
          filter: `grayscale(${selectedId === ottId ? 0 : 100}%)`
          p: 1,
          m: 1,
        }}
```
아직 제가 sx props에 대해서 잘 알지 못해서 조심스럽지만 이런식으로는 하면 makeStyle을 사용하지 않아도 되지 않을까 싶어서 제안 드려봅니다.

Reply:
sx에서 filter 속성 적용이 안되는줄 알고 makeStyle을 사용하였는데 
훨씬 코드가 간결해지겠네요 :)
감사합니다 도르!